lib/floe/workflow/states/task.rb

```diff
@@ -118,12 +119,14 @@ def catch_error!(error)
 
           context.next_state = catcher.next
           context.output     = catcher.result_path.set(context.input, error)
+          logger.info("Running state: [#{long_name}] with input [#{context.input}]...CatchError - next state: [#{context.next_state || "END"}] output: [#{context.output}]")
```

Comment:
I thought we decided on "END" not being presented this way?  (think you wanted to say "workflow complete" or something?)

Reply:
ugh. must have lost it via rebase. I had just made it so much better. ugh